### DISCUSSION TITLE
目录的生成功能为什么取消了？

### DISCUSSION THREAD

**Discussion by winq2008** [yutto-dev/yutto#130] [Q&A] [answered]
目录的生成功能以前不是挺好用的吗。生成一个目录文件，再搭配mpv播放器，自动按列表顺序播放视频集很爽的啊？
能不能把这个功能加回来？

**Comment by SigureMo** [comment, marked as answer]
是说播放列表吧，可以通过参数 `--subpath-template` 来修改生成视频的目录结构/文件名，利用 `id` 变量来维持有序性

yutto 不会添加播放列表功能的
